webviz_subsurface/plugins/_well_completions.py

```diff
@@ -0,0 +1,462 @@
+    ):
+        super().__init__()
+        self.theme = webviz_settings.theme
+        self.colors = self.theme.plotly_theme["layout"]["colorway"]
+        self.compdat_file = compdat_file
+        self.zone_layer_mapping_file = zone_layer_mapping_file
+        self.well_attributes_file = well_attributes_file
+        self.ensembles = ensembles
+        self.ens_paths = {
+            ens: webviz_settings.shared_settings["scratch_ensembles"][ens]
+            for ens in ensembles
+        }
+
+        self.set_callbacks(app)
+
+    def add_webvizstore(self):
+        return [
+            (
+                create_ensemble_dataset,
+                [
+                    {
+                        "ensemble": ensemble,
+                        "ensemble_path": self.ens_paths[ensemble],
+                        "compdat_file": self.compdat_file,
+                        "zone_layer_mapping_file": self.zone_layer_mapping_file,
+                        "well_attributes_file": self.well_attributes_file,
+                        "colors": self.colors,
+                    }
+                    for ensemble in self.ensembles
+                ],
+            ),
+        ]
+
+    @property
+    def layout(self) -> html.Div:
+        return html.Div(
+            children=[
+                wcc.FlexBox(
+                    children=[
+                        html.Div(
+                            style={"flex": "1", "padding": "10px"},
+                            children=html.Label(
+                                children=[
+                                    html.Span(
+                                        "Ensemble", style={"font-weight": "bold"}
+                                    ),
+                                    dcc.Dropdown(
+                                        id=self.uuid("ensemble_dropdown"),
+                                        options=[
+                                            {"label": ens, "value": ens}
+                                            for ens in self.ensembles
+                                        ],
+                                        clearable=False,
+                                        value=self.ensembles[0],
+                                        persistence=True,
+                                        persistence_type="session",
+                                    ),
+                                ]
+                            ),
+                        ),
+                        html.Div(style={"flex": 4}),
+                    ],
+                ),
+                html.Div(id=self.uuid("well_completions_wrapper"),),
+            ]
+        )
+
+    def set_callbacks(self, app):
+        @app.callback(
+            [
+                Output(self.uuid("well_completions_wrapper"), "children"),
+                Output(self.uuid("well_completions_wrapper"), "style"),
+            ],
+            [Input(self.uuid("ensemble_dropdown"), "value"),],
```

Comment:
```suggestion
            Output(self.uuid("well_completions_wrapper"), "children"),
            Output(self.uuid("well_completions_wrapper"), "style"),
            Input(self.uuid("ensemble_dropdown"), "value"),
```
Both works. In new plugins we try/aim for using the new Dash syntax.

Reply:
Ok!